models/Tool/SettingsStore.php

```diff
@@ -158,7 +159,7 @@ public function getScope(): ?string
      */
     public function setScope(?string $scope): void
```

Comment:
The `@param` annotation is wrong and must be `string|null` (can't comment on that line).